packages/embed/src/factories/create-popover/notification-days.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { canBuildNotificationDot } from './notification-days'
+
+const setItemMock = jest.fn()
+
+const localStorageMockhelper = (value: any) => {
+  Object.defineProperty(window, 'localStorage', {
+    value: {
+      getItem: jest.fn().mockReturnValue(JSON.stringify(value)),
+      setItem: setItemMock,
+    },
+  })
+}
+
+describe('notification-days', () => {
+  describe('#canBuildNotificationDot', () => {
+    it('should return true when expiry date is not set in local storage and set it in local storage', () => {
+      const data = {
+        otherFormId: {
+          expiryDate: 10,
+        },
+      }
+      localStorageMockhelper(data)
+
+      expect(canBuildNotificationDot('myFormId', 2)).toBeTruthy()
+      expect(setItemMock.mock.calls[0][0]).toBe('tfNotificationData')
```

Comment:
```suggestion
      expect(setItemMock).toHaveBeenCalledWith('tfNotificationData')
```

Reply:
It is called with 2 args and 2nd one needs to be checked via regex.